xrootd.sh

```diff
@@ -15,7 +15,7 @@ build_requires:
 ---
 #!/bin/bash -e
 [[ -e $SOURCEDIR/bindings ]] && XROOTD_V4=True && XROOTD_PYTHON=True || XROOTD_PYTHON=False
-PYTHON_EXECUTABLE=$( $(realpath $(which python3)) -c 'import sys; print(sys.executable)')
+PYTHON_EXECUTABLE=$( $(readlink -f $(which python3)) -c 'import sys; print(sys.executable)')
```

Comment:
This will not work on mac.

Reply:
Ah, I see. somehow the Mac does not like the -f option. Checking again, I see that it is not really needed; I will update the pull request